### PR TITLE
Add Feature - Add "enable_ipam" option for private_network

### PR DESCRIPTION
**Pull request by jgalais**
- Add "enable_ipam" option in "private_network" variable
- Add Output to retrieve database instance IP
- Add "ip_net" and "enable_ipam" in "private_network" as optional because we must use one and not both
- Format documentation with tfdocs

### CODE REVIEW

File: outputs.tf [hunk 1]
```diff
@@ -8,6 +8,12 @@ output "instance_id" {
   value       = scaleway_rdb_instance.this.id
 }
 
+output "instance_ip" {
+  description = "Database instance IP"
+  sensitive   = true
```
Comment (benoit-garcia) [inline outputs.tf]: I don't see the point of having the instance IP marked as sensitive?

**Review by benoit-garcia — CHANGES_REQUESTED** [review]
Thanks for your contribution @jgalais !
However there's an issue with the documentation: It looks like parameters `settings.escape` and `sort.by` are not properly set on your side. Terraform-docs should automatically uses the configuration file at the top of the repository but you can set it on the command line by running the following command:
```bash
terraform-docs -c .terraform-docs.yaml ./
```
If it helps I'm using the latest version of terraform-docs ([v0.17.0](https://github.com/terraform-docs/terraform-docs/releases/tag/v0.17.0)) on my side.

File: outputs.tf (hunk 1 above)
Comment (jgalais) [reply outputs.tf]: You're right it's a mistake on my part.
I fixed that.

**Comment by jgalais** [comment]
I fixed terraform docs.
I forgot to use you're configuration file.

**Review by benoit-garcia — APPROVED** [review]

**Comment by benoit-garcia** [comment]
@jgalais Thanks!
It has been approved, merged and released

**Pull request merged**